designsafe/static/styles/bootstrap3.breadcrumb.css

```diff
@@ -1,4 +1,4 @@
-@import url("https://cdn.jsdelivr.net/gh/TACC/Core-Styles@v2.27.0/dist/components/bootstrap3.breadcrumb.css");
+@import url("https://cdn.jsdelivr.net/gh/TACC/Core-Styles@v2.37.2/dist/components/bootstrap3.breadcrumb.css");
```

Comment:
[No changes to `bootstrap3.breadcrumb.css` since 2.27.0.](https://github.com/TACC/Core-Styles/commit/b3d1227)